### PR TITLE
Vishal/mainnet6 updates

### PR DESCRIPTION
**Pull request by vishalchangrani**
Closes: #???

## Description

Updates for mainnet6

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: nodeoperators/nodelist.json [hunk 1]
```diff
@@ -2577,7 +2577,7 @@
     },
     {
         "Role": "execution",
-        "Address": "139.178.85.161:3569",
+        "Address": "flow-execution.samsungnext.com:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2580]: samsung change - ip to hostname

File: nodeoperators/nodelist.json [hunk 2]
```diff
@@ -2822,5 +2822,37 @@
         "Stake": 1000,
         "NetworkPubKey": "4330dacae96a2a88cd52d5d040358e1c32d9ad0ba7cd62316ed2390a9d1b2d6802b3e880b5755f1a460638eb3129d116eebdeaa1ffbc30ef8e66707e3353e006",
         "StakingPubKey": "9727ecb75b071a77613f4e8ef726aefb09bc467aff1f3818c84ef21aeb587876cca7d1a3ee3a56395090f8582f8f6f9a017c6984e783627712f4a9db2a87ca11e3ccb8c2eea803ba60be790c61691324b8dfd52a9c8608ccdb096ada59c4b42c"
+    },
+    {
+        "Role": "access",
+        "Address": "ec2-52-202-146-129.compute-1.amazonaws.com:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2828]: alchemy

File: nodeoperators/nodelist.json [hunk 3]
```diff
@@ -2822,5 +2822,37 @@
...
+        "NodeID": "688e4ea5334117b578ea8b55c760af6207c186a71fbe2e2d7bcf837676b2b985",
+        "Stake": 0,
+        "NetworkPubKey": "b41b88439a9d1fc025ddd36c0689cd37944490115740a1ce523cd6c5df14f78145442aa05317d0893d2598931886ed5f7d3248c29f5748db8b2d55d2f60b4523",
+        "StakingPubKey": "b75cba817b162b0ac6dfc1fbecbd489b22218f27bdc3df21b79db643a91f27934e0c4dce06ccca902c1977ced11018dd0349d8b7e2da2045a57fa66e6adb9965db8aa0f6df4722f2f8940a75a87307ea809fcc7c8d02ccf79e8a74f4875cf8e5"
+    },
+    {
+        "Role": "access",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2835]: blockwatch

File: nodeoperators/nodelist.json [hunk 4]
```diff
@@ -2822,5 +2822,37 @@
...
+        "Address": "flow.blockwatch.cc:3569",
+        "NodeID": "a2168ec28f92e405b291290c345b925225a686cb24541dce8f09e8808d8d6527",
+        "Stake": 0,
+        "NetworkPubKey": "5ac4b3985bfb77fdf2b97d544dc2d7119d0d47e30642784c61ef88305cff99f1b02f2f16e8e804465a9b89d8ca38a7df881698f68c097976744721b0ff3f7026",
+        "StakingPubKey": "91df2d9b6813bce78c892e17b575ac6bd09a5f31905282d6df55282c8f38005fb6f747330eb9d9c05ff0aec03f1a85b60fab9965a73a1105c7d744fa467230def030d88fdccc664db75cac028dc9da60f5b124a17e0569bc37815c7da2d8132a"
+    },
+    {
+        "Role": "access",
+        "Address": "flow-mainnet-access-node.metrika.co:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2844]: metrika

File: nodeoperators/nodelist.json [hunk 5]
```diff
@@ -2822,5 +2822,37 @@
...
+        "NodeID": "53241f85deb9df95f9bb04b2dd46ce225fbca6540050d3d40088243f02a6019c",
+        "Stake": 0,
+        "NetworkPubKey": "c8b80bfe863c81412e1d2c5bf5d4aa8a8629c7cd64473d23aa74f2209ba86b2b53e85030389b9a4bba3bba511a99b2e52bc1d9ac6ae4bc854a454e06a96d2a85",
+        "StakingPubKey": "85ae992114b5cebf9ba94b8723348c5a03dc2ef71c64027dc0c3803eda1f5b1d9ff915e572613f15572421163b1fee01047bec49d608eec6c96e3459a7de198f8a4fdf06e3d573a09a65a179c437dd2f3732c41a3d1c7eeccfd7ca0395458691"
+    },
+    {
+        "Role": "access",
+        "Address": "node1.nonfungible.com:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2852]: nonfungible

File: nodeoperators/nodelist.json [hunk 6]
```diff
@@ -2822,5 +2822,45 @@
...
+        "NodeID": "6500b3a162c8a56d15b41269bdeaf5cf90a5830f350fe72330f6940c7ead66b4",
+        "Stake": 0,
+        "NetworkPubKey": "e8222eab1c1a6449984a2b9874a9382e1953ed7566494af28f0ca70f3b2e498a0cf2a890eff911d9a71a0e961f60182dbd801193d629e54b1df8fec4e2c0ec3a",
+        "StakingPubKey": "a7174b001f7c0457e3ded1f724595cff8765bc9de7db3c46e68d79c0b44e328c4e350821d433d1e058c839fc50ef36cb1749b1952d128a5e26487da028e418891e595267d295c9df4980bfd3055e6b8fd7526fa13bbb5ff667eef88642ea155e"
+    },
+    {
+        "Role": "verification",
+        "Address": "52.53.128.190:3569",
```
Comment (Kay-Zee) [inline nodeoperators/nodelist.json:2860]: Houbi

File: nodeoperators/nodelist.json [hunk 7]
```diff
@@ -2553,19 +2553,19 @@
...
-        "NetworkPubKey": "e922715dae03ced2028d184a68ea80fe744a0fa215362b82f8d5b9413449e0254c736c260aae90d44bb78f80e6437f3d7707ebd08ca02845372152c0810df0ec",
-        "StakingPubKey": "a58f7ef25ee14db08e69be4f04ebe3c444c3b216ecf8d536bdb7e9912b1a4a47c7eb5b101ba1ae99c2b132fcbf4ae68c116d462487e4557e7663d19ec642a7fb3bb0ea53c8f037da850e276b2ec55799ad69445dcfe00a15afd0577918f99c2c"
+        "NetworkPubKey": "7680eb3035bcb46d92a7fad079266b0f151cee180c54506207998ea90f02214347e200b431a37d1c17c80bde3b99748954a050256af502ba8f1453c4fd466bff",
+        "StakingPubKey": "aa5c8f996d0bf1b82a7d58f760dad068216c7c863daf3ee51e19063208f67054e058db07c0a9e1e41affb0ef86b4aeb61586270d2ef4cec4a551a9887272de8e19c37dbad8ceb20fe9de4f155eaea61fc6a5cac4a47016f2fc0b32aec463d2af"
     },
     {
         "Role": "collection",
-        "Address": "203.132.22.24:3569",
-        "NodeID": "dc0fb2d914f0c12cc1de527d22974b8f2de376e4765e084e0b8be14b51b5ef81",
+        "Address": "185.38.22.188:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:2564]: ubisoft

**Review by Kay-Zee — APPROVED** [review]

**Pull request merged**